common/web/input-processor/src/text/prediction/languageProcessor.ts

```diff
@@ -66,6 +66,13 @@ interface LanguageProcessorEventMap {
   'statechange': StateChangeHandler,
   'tryaccept': TryUIHandler,
   'tryrevert': () => void,
+
+  /**
+   * Is called synchronously as part of suggestion application when successful.
```

Comment:
When exactly is this called; for example, it'd be helpful to know whether the text buffer in the app has been updated

Reply:
Is this more clear?

```suggestion
   * Is called synchronously once suggestion application is successful and the context has been updated.
```

Reply:
Yep, looks good.